arelle/plugin/loadFromOIM.py

```diff
@@ -1560,21 +1572,30 @@ def dimChecks():
                                         if dimName not in checkedDims:
                                             dimValue = inheritedDims[dimName]
                                             # resolve column-relative dimensions
-                                            if isinstance(dimValue, str) and dimValue.startswith("$"):
-                                                dimValue = dimValue[1:]
-                                                if not dimValue.startswith("$"):
-                                                    dimValue, _sep, dimAttr = dimValue.partition("@")
-                                                    if _sep and dimAttr not in ("start", "end"):
-                                                        hasHeaderError = True
-                                                        error("xbrlce:invalidPeriodSpecifier",
-                                                              _("Table %(table)s %(source)s %(dimension)s period-specifier invalid: %(target)s, url: %(url)s"),
-                                                              table=tableId, source=dimSource, dimension=dimName, target=dimAttr, url=tableUrl)
-                                                    if dimValue not in checkedParams:
-                                                        checkedParams.add(dimValue)
-                                                        if dimValue in ("rowNumber", ) or (dimValue in header and dimValue not in commentColumns) or dimValue in tableParameters or dimValue in reportParameters:
-                                                            checkedDims.add(dimValue)
-                                                        else:
-                                                            potentialInvalidReferenceTargets[dimName] = dimValue
+                                            if isinstance(dimValue, str):
+                                                if dimValue.startswith("$"):
```

Comment:
Can you explain some of these cases? What does it mean to have:

- $ (looks like maybe a date?)
- $$
- text:text (looks like a dimension:member pairing?)